recipes/coinmp/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "CoinMP" %}
+{% set version = "1.8.3" %}
+{% set sha256 = "7ae0c37c15d09f91c7fda33729d930e612ded8120d96c3e0932d82b7127cc5d0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: http://www.coin-or.org/download/source/CoinMP/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - pkgconfig
+    - readline
+    - zlib
+    - bzip2
+  run:
+    - readline
+    - zlib
+    - bzip2
```

Comment:
So these need to be pinned according to this [list]( https://github.com/conda-forge/conda-forge.github.io/blob/af6e7b282efb2bf9b62fe10f8093d76b37c333e1/scripts/pin_the_slow_way.py#L29-L60 ). Please update in the feedstock and bump the build number.